Services/DataX.Flow/DataX.Flow.Common/Helper.cs

```diff
@@ -56,7 +56,7 @@ public static string GetSecretFromKeyvaultIfNeeded(string value)
         /// <returns>true if it is a secret, otherwise false</returns>
         public static bool IsKeyVault(string value)
         {
-            return (value.StartsWith(GetKeyValutNamePrefix()) || value.StartsWith(GetSecretScopePrefix()));
+            return value.StartsWith(GetKeyValutNamePrefix()) || value.StartsWith(GetSecretScopePrefix());
```

Comment:
why change this one?

Reply:
I was seeing a formating error, hence fixing it in this checkin